met-web/src/components/common/dateHelper.tsx

```diff
@@ -1,17 +1,30 @@
-import dayjs from 'dayjs';
-import { format, utcToZonedTime } from 'date-fns-tz';
-import utc from 'dayjs/plugin/utc';
-import timezone from 'dayjs/plugin/timezone';
+import { format, utcToZonedTime, zonedTimeToUtc } from 'date-fns-tz';
+import moment from 'moment';
 
-dayjs.extend(utc);
-dayjs.extend(timezone);
-
-const formatInTimeZone = (date: string, fmt: string, tz: string) =>
+const formatToPSTTimeZone = (date: string, fmt: string, tz: string) =>
     format(utcToZonedTime(date, tz), fmt, { timeZone: tz });
 
-export const formatDate = (d: string, formatString = 'yyyy-MM-dd') => {
-    if (d) {
-        return formatInTimeZone(d, formatString, 'UTC');
+const formatToUTCTimeZone = (date: string, fmt: string, tz: string) =>
+    moment(
+        zonedTimeToUtc(date, tz).toISOString().substring(0, 10) +
+            ' ' +
+            zonedTimeToUtc(date, tz).toISOString().substring(11, 19),
+    ).format(fmt);
+
+export const formatDate = (date: string, formatString = 'yyyy-MM-dd') => {
+    if (date) {
+        return formatToPSTTimeZone(date.substring(0, 10) + 'T' + date.substring(11, 25) + 'Z', formatString, 'PST');
+    } else {
+        return '';
+    }
+};
+
+export const formatToUTC = (date: string, formatString = 'yyyy-MM-dd') => {
+    if (date) {
+        const utcDate = formatToUTCTimeZone(date, formatString, 'PST');
+        console.log(zonedTimeToUtc(date, 'PST').toISOString());
+        console.log(utcDate);
+        return utcDate.substring(0, 10) + ' ' + utcDate.substring(11, 25);
```

Comment:
it seems odd that we need substrings to get the date, also shouldn't it follow the format specified in t he formatString param?

Reply:
agreed, its not needed here so I removed it.